### ISSUE TITLE
[REQ] Buds3 tray icon

### ISSUE THREAD

**Issue by mauriciomlima** [timschneeb/GalaxyBudsClient#542]
What about displaying Buds3 tray icon when Buds3/Buds3 Pro is connected???

**Comment by akano12** [comment]
i know this isn't related to your request, does any of the module tests pass on your buds 3 pro? Mine pass only 3-4 tests so i'm not sure if the manager isn't fully compatible with the buds 3 or there's something wrong in my buds

**Comment by yugthebug33** [comment]
yeah i experience that too @akano12 but im fairly sure the client knows what flavour of buds you are using so it would be nice to have a matching tray icon
